src/components/LayerUI.tsx

```diff
@@ -358,6 +366,10 @@ const LayerUI = ({
   );
 
   const renderExportDialog = () => {
+    if (canvasActions && canvasActions.export === false) {
```

Comment:
```suggestion
    if (!canvasActions.export === false) {
```
The current check can lead to confusion, for example if host doesn't pass `export: false` we will show it but the behaviour should be default props all are set to true and if user pass this prop then that gets the preference so then if and only if the action name is true we show else we hide.